src/measure/OSArgument.cpp

```diff
@@ -763,7 +763,7 @@ namespace measure {
       return false;
     }
 
-    double minValue = std::numeric_limits<double>::min();
+    double minValue = std::numeric_limits<double>::lowest();
```

Comment:
Using `min()` here means using the smallest positive value. NOT what was expected.

Use `lowest()` which equals `-max()`

cf https://godbolt.org/z/K1aq87Yjh

```
std::numeric_limits<double>::max()   =  1.7976931348623157e+308
std::numeric_limits<double>::min()   =  2.2250738585072014e-308  # POSITIVE minimum finite value
std::numeric_limits<double>::lowest()= -1.7976931348623157e+308
```